src/StacBrowser.vue

```diff
@@ -137,8 +146,32 @@ export default {
     }
   },
   created() {
-    this.$router.onReady(() => this.parseQuery(this.$route));
-    this.$router.afterEach(to => this.parseQuery(to));
+    let unwatch = null;
```

Comment:
Could you add some comments about what the new lines do here? Especially the watchers? It's not 100% obvious...

Having stateQueryParameters back in localDefaults is still an issue, right? It would be so much cleaner if it would just work through that mechanism...

Reply:
> Having stateQueryParameters back in localDefaults is still an issue, right? It would be so much cleaner if it would just work through that mechanism...

You are welcome to try and make it work but I don't seem to be able to sorry.

Reply:
Thanks for the update.